### PR TITLE
Custom environment

### PR DESCRIPTION
**Pull request by RasmusBrostroem**
solve #4 

solve #3

### CODE REVIEW

File: gym_game/envs/connectFour.py [hunk 1]
```diff
@@ -2,13 +2,19 @@
 import numpy as np 
 
 class connect_four():
-    def __init__(self, game_size: int):
+    def __init__(self):
         # The attributes to display the game
-        self.size = game_size
+        self.size = 1000
```
Comment (jbirkesteen) [inline gym_game/envs/connectFour.py:7]: What is the reasoning behind this change? Do you even use this attribute for anything?

File: training.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+# Environment
+env = gym.make('ConnectFour-v0')
+
+s = env.reset()
+env.configurePlayer(random.choice([-1,1]))
+
+# Main training loop
+# for gen in range(generations):
+#     model_save_path = 'AgentParameters/agent_params'+gen+'.pkl'
```
Comment (jbirkesteen) [inline training.py:41]: Check out docs for `torch_save()` if you don't know about these. I think they would be the easiest way of saving model parameters (as well as perhaps other data)

**Review by jbirkesteen — APPROVED** [review]
We've already went through most of these changes together, and everything looks fine (except for what I know you've already changed locally in later versions). However, 1 change confused me a little bit. I guess it shouldn't be an issue, though, so if not, take this as an approval.
Also, see my comment for saving. We can discuss this later

File: training.py (hunk 2 above)
Comment (jbirkesteen) [reply training.py:41]: See second section in #10.

File: gym_game/envs/connectFour.py (hunk 1 above)
Comment (RasmusBrostroem) [reply gym_game/envs/connectFour.py:7]: Use it for the size of the window, when we want to display the game being played.

Changed it so it is not a parameter you can give to the game, but rather something we decide inside the game object

**Pull request merged**